src/libraries/System.Drawing.Common/src/System/Drawing/Drawing2D/LinearGradientBrush.cs

```diff
@@ -232,14 +232,16 @@ public bool GammaCorrection
                     throw new NullReferenceException();
 
                 if (value.Positions == null)
+#pragma warning disable CA2208 // Instantiate argument exceptions correctly
                     throw new ArgumentNullException("source");
+#pragma warning restore CA2208
```

Comment:
Why is "source" correct here?

Reply:
I tought it is for compatability, from the comment on row 230 `This is the original behavior on Desktop .NET`

Reply:
updated with `throw new ArgumenException(SR.Format(SR.InvalidArgumentValue, "value.Positions", value.Positions), nameof(value));`